nifi-marklogic-processors/src/main/java/org/apache/nifi/marklogic/processor/ExecuteScriptMarkLogic.java

```diff
@@ -241,9 +258,22 @@ public final void onTrigger(final ProcessContext context, final ProcessSessionFa
                 }
             }
 
+            String scriptBody = determineScriptBody(context, originalFF);
```

Comment:
There's already logic in `buildCall` to determine if the user is using a script or a module. Should just modify that to add the appropriate attribute to the original FlowFile.